frigate/http.py

```diff
@@ -651,8 +652,15 @@ def latest_frame(camera_name):
         frame = current_app.detected_frames_processor.get_current_frame(
             camera_name, draw_options
         )
-        if frame is None:
-            frame = np.zeros((720, 1280, 3), np.uint8)
+
+        if frame is None or datetime.now().timestamp() > (
+            current_app.detected_frames_processor.get_current_frame_time(camera_name)
+            + 10
+        ):
+            error_image = glob.glob("/opt/frigate/frigate/images/camera-error.jpg")
```

Comment:
Maybe an idea for the future would be to generate images programmatically so you can embed the error cause.

https://towardsdatascience.com/adding-text-on-image-using-python-2f5bf61bf448

Reply:
Yeah, we considered that but don't want to cause issues for the integration or frigate card which aren't prepared to handle that so a static image is the solution for now.